Shapes/Shape.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Shapes
+{
+    /// <summary>
+    /// interface of shape
+    /// </summary>
+    internal interface Shape
+    {
```

Comment:
Having an id for each shape is best practice. You can access specific shapes without passing an object. For instance, deleting a shape from your list requires only the shape's id and not its other properties and functions.